apps/deploy-web/src/queries/useTemplateQuery.tsx

```diff
@@ -1,14 +1,15 @@
-import { QueryKey, useMutation, useQuery, useQueryClient, UseQueryOptions } from "react-query";
 import { Snackbar } from "@akashnetwork/ui/components";
 import axios from "axios";
 import { useRouter } from "next/navigation";
 import { useSnackbar } from "notistack";
+import { QueryKey, useMutation, useQuery, useQueryClient, UseQueryOptions, UseQueryResult } from "react-query";
 
 import { useCustomUser } from "@src/hooks/useCustomUser";
+import { services } from "@src/services/http/http-browser.service";
 import { ITemplate } from "@src/types";
-import { ApiUrlService } from "@src/utils/apiUtils";
 import { UrlService } from "@src/utils/urlUtils";
 import { QueryKeys } from "./queryKeys";
+import { TemplateCategory, TemplateOutputSummary } from "@akashnetwork/http-sdk/src/template/template-http.service";
```

Comment:
would this be fine?
```suggestion
import type { TemplateCategory, TemplateOutputSummary } from "@akashnetwork/http-sdk";
```

Reply:
one more thing for eslint to check. I use autoimporting a lot and do not review what actually was inserted by IDE.

will fix this one, it additionally requires exporting of template service from  `@akashnetwork/http-sdk`